packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -319,6 +319,12 @@ export function getReqSerializers(config: ChainForkConfig): ReqSerializers<Api,
     fromJson: (data) => getSignedBlindedBeaconBlockType(data as allForks.SignedBlindedBeaconBlock).fromJson(data),
   };
 
+  function extractSlot(signedBlockOrContents: allForks.SignedBeaconBlockOrContents): Slot {
+    return isSignedBlockContents(signedBlockOrContents)
```

Comment:
this logic is only for (full) signed block and not blinded block, in case of blinded block the slot is always accessible via `data.message.slot` even post deneb